src/steps/Microsoft.DotNet.UpgradeAssistant.Steps.Packages/PackageUpdaterPreTFMStep.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.DotNet.UpgradeAssistant.Steps.Packages
+{
+    public class PackageUpdaterPreTFMStep : PackageUpdaterStep
+    {
+        public override string Description => "Update package references and remove transitive dependencies";
+
+        public override string Title => "Clean up NuGet package references";
+
+        public override IEnumerable<string> DependsOn { get; } = new[]
+        {
+            // Project should be backed up before changing package references
+            "Microsoft.DotNet.UpgradeAssistant.Steps.Backup.BackupStep",
```

Comment:
Do you need to explicitly state this?

Reply:
Its stated explicitly in the PackageUpdaterStep and in other steps.

Reply:
Removed backup step, its working with out that too